README.md

```diff
@@ -99,10 +99,10 @@ mod ffi {
         pub fn reset(mut self: Pin<&mut Self>) {
             let data: DataSerde = serde_json::from_str(DEFAULT_STR).unwrap();
             self.as_mut().set_number(data.number);
-            self.as_mut().set_string(&QString::from_str(&data.string));
+            self.as_mut().set_string(QString::from_str(&data.string));
         }
 
-        #[qinvokable]
+        #[qinvokable(return_cxx_type = "QString")]
```

Comment:
Do we also have a way to also declare the parameter C++ types? :thinking:

Reply:
So far we don't need it, as you can do references eg `&QVariant`. Maybe later we might need something.